charts/osm/values.yaml

```diff
@@ -33,6 +33,7 @@ OpenServiceMesh:
   enableBackpressureExperimental: false
   enableEgress: false
   enableMetricsStack: true
+  enableGrafana: true
```

Comment:
Now that we have a separate flag for grafana the `enableMetricsStack` is primarily used for Prometheus deployment, maybe we should change that flag as well to make it more explicit 

@eduser25 as FYI

Reply:
+1, agree with Sneha, at this point let's have independant toggles for Grafana / Prometheus deployments

Reply:
Ok, sounds good!